modules/swagger-codegen/src/main/resources/TypeScript-Fetch/api.mustache

```diff
@@ -90,11 +90,12 @@ export const {{classname}}FetchParamCreator = {
             .replace(`{${"{{baseName}}"}}`, `${ params["{{paramName}}"] }`){{/pathParams}};
         let urlObj = url.parse(baseUrl, true);
 {{#hasQueryParams}}
-        urlObj.query = {{#supportsES6}}Object.{{/supportsES6}}assign({}, urlObj.query, {
-            {{#queryParams}}
-            "{{baseName}}": params["{{paramName}}"],
-            {{/queryParams}}
-        });
+    urlObj.query =  {{#supportsES6}}Object.{{/supportsES6}}assign({}, urlObj.query)
```

Comment:
Could you please fix the indentation of urlObj.query and then regenerate the samples? Thanks :-)